fetch/stale-while-revalidate/frame-removal.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test that frame removal cancels its revalidations</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<body>
+<script>
+
+const wait_for_message = () => {
+  return new Promise((resolve, reject) => {
+    window.addEventListener('message', e => {
+      resolve(e);
+    });
+  });
+};
+
+async_test(async function(t) {
+  let frame = document.createElement("iframe");
+  frame.src = "resources/frame-removal.html";
+  frame.onload = t.step_func(async () => {
```

Comment:
There is a routine with_iframe that does the loading of the iframe.
That would allow to change it to a promise test, which I usually prefer over async_test.